<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSwitherConfigureClass</class>
 <widget class="QDialog" name="CSwitherConfigureClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>655</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>路由器属性</string>
  </property>
  <property name="toolTip">
   <string>路由器属性</string>
  </property>
  <property name="statusTip">
   <string>路由器属性</string>
  </property>
  <property name="whatsThis">
   <string>路由器属性</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" rowstretch="0">
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>20</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>属性</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>路由器名称：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="deviceNameLineEdit"/>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>网络位置：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="deviceLocation">
                <property name="editable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>域ID：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="id"/>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>端口数目：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="portNumComboBox">
                <property name="editable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>协议栈：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="protocolComboBox">
                <property name="editable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>端口连接列表</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QTableWidget" name="tableWidget">
            <property name="minimumSize">
             <size>
              <width>607</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>607</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="autoScrollMargin">
             <number>16</number>
            </property>
            <property name="columnCount">
             <number>6</number>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>100</number>
            </attribute>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>56</number>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>30</number>
            </attribute>
            <attribute name="verticalHeaderHighlightSections">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderMinimumSectionSize">
             <number>21</number>
            </attribute>
            <column>
             <property name="text">
              <string>源端口</string>
             </property>
             <property name="font">
              <font>
               <kerning>true</kerning>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>源IP</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>目的端</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>目的IP</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>链路类型</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>链路速率</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="okButton">
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
